File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/post/PostFormValidator.java
package com.isystk.sample.web.admin.controller.html.post;

import com.isystk.sample.common.validator.AbstractValidator;
import org.springframework.stereotype.Component;
import org.springframework.validation.Errors;

/**
 * 投稿 入力チェック
 */
@Component
public class PostFormValidator extends AbstractValidator<PostSearchForm> {

  @Override
  protected void doValidate(PostSearchForm form, Errors errors) {

  }
}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/post/PostFormValidator.kt
package com.isystk.sample.web.admin.controller.html.post

import com.isystk.sample.common.validator.AbstractValidator
import org.springframework.stereotype.Component
import org.springframework.validation.Errors

/**
 * 投稿 入力チェック
 */
@Component
class PostFormValidator : AbstractValidator<PostSearchForm?>() {
    override fun doValidate(form: PostSearchForm?, errors: Errors?) {
    }
}